app/src/main/java/uk/gov/onelogin/ui/profile/ProfileScreen.kt

```diff
@@ -0,0 +1,47 @@
+package uk.gov.onelogin.ui.profile
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import uk.gov.android.onelogin.R
+import uk.gov.android.ui.pages.TitledPage
+import uk.gov.android.ui.pages.TitledPageParameters
+
+@Composable
+fun ProfileScreen() {
+    TitledPage(
+        parameters = TitledPageParameters(
+            R.string.app_profile
+        ) {
+            HorizontalDivider()
```

Comment:
Could probably build this email display section into a separate composable given it's used twice...

Reply:
That's what I was thinking as well, but then I realised it's mostly one component? (a formatter Text field, surrounded by dividers).
And I've also struggled to come up with a name that is not `strategic app specific`, so it would make sense to have it in the Design System Library. `TwoLineHeader`?

Reply:
I don't think it needs to be in the UI repo, I think you can call it some thing like `EmailHeader` and keep it somewhere in this repo

Reply:
yeah, that makes sense. We need `core/ui`

Reply:
done d10d7669d24ddcb7138b0ba32ae189aec6979ade